src/local-peers.js

```diff
@@ -186,6 +224,53 @@ class Peer {
     await onDrain.promise
   }
 
+  /**
+   * Check if RPC Acknowledgement messages are supported by this peer
+   * @returns {boolean}
+   */
+  supportsAck() {
+    return this.#features?.includes(DeviceInfo_RPCFeatures.ack) ?? false
```

Comment:
is the optional chaining necessary here?

```suggestion
    return this.#features.includes(DeviceInfo_RPCFeatures.ack) ?? false
```

Reply:
Yes because the field is optional in the device Info

Reply:
Good point, it will never be undefined